src/pet.js

```diff
@@ -0,0 +1,83 @@
+const MAXIMUM_FITNESS = 10;
+const minHungerLevel = 0;
+
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = minHungerLevel;
+    this.fitness = MAXIMUM_FITNESS;
+    this.children = [];
+};
+
+Pet.prototype = {
+    get isAlive() {
+      return this.age < 30 && this.hunger < 10 && this.fitness > 0;
+    }
+  };
```

Comment:
Indentation is a bit out here, you should have 4 spaces for each layer here to keep things consistent and the closing bracket should be at the same level as the opening of the function:
```
Pet.prototype = {
    get isAlive() {
        return this.age < 30 && this.hunger < 10 && this.fitness > 0;
    }
};
```